### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by bbrewer-forge**

### CODE REVIEW

**Comment by kcarra** [comment]
:pray: :pray:

File: scripts/wdc-test.sh [hunk 1]
```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
-
-source ./scripts/lib_common.sh
```
Comment (bbrewer-forge) [inline scripts/wdc-test.sh:5]: will restore

File: .github/workflows/andi.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
+name: Andi
+on:
+  push:
+    branches:
+      - master
+    paths:
+    - 'apps/andi/**'
```
Comment (bbrewer-forge) [inline .github/workflows/andi.yml:7]: pros: allows us to truly skip steps (not spin up a vm for 45 seconds to check if it changed)
cons: lots of copy and paste for library based apps we want to check

Comment (LtChae) [reply .github/workflows/andi.yml:7]: Adding new lines here isn't too horrible a cost to pay in my opinion

**Review by LtChae — APPROVED** [review]

**Pull request merged**